.github/scripts/end2end/run-e2e-ctst.sh

```diff
@@ -146,4 +157,4 @@ kubectl run $POD_NAME \
       }
     ]
   }
-}' -- ./run "$COMMAND" $WORLD_PARAMETERS --parallel $PARALLEL_RUNS --retry $RETRIES --retry-tag-filter @Flaky --format junit:$JUNIT_REPORT_PATH
+}' -- ./run "$COMMAND" $WORLD_PARAMETERS --parallel $PARALLEL_RUNS --retry $RETRIES "$@" --retry-tag-filter @Flaky --format junit:$JUNIT_REPORT_PATH 
```

Comment:
nit: may be best to put `"$@"` at the end, so it may override params already provided